charts/snapshot-controller/Chart.yaml

```diff
@@ -10,9 +10,12 @@ maintainers:
 description: |
   Deploys a Snapshot Controller in a cluster. Snapshot Controllers are often bundled with the Kubernetes distribution,
   this chart is meant for cases where it is not.
+  Also deploys the Snapshot Validation Webhook and configures your cluster to validate every `VolumeSnapshot` and
+  `VolumeSnapshotContent` resource by sending it to the webhook.
 keywords:
   - storage
   - snapshot
+  - validation
```

Comment:
There is a `annotations` section at the end of this file which needs to be updated: we no longer need to recommend the webhook chart.